src/models/index.tsx

```diff
@@ -15,3 +15,8 @@ export enum Theme {
   dark,
   light,
 }
+
+export interface FavoriteMovie {
+  movieId: number;
+}
```

Comment:
I dont see any reason why we need this to be interface. Seems like FavoritesBtnProps type will be enough

Reply:
Thanks for a hint. Did you mean just to use a type FavoritesBtnProps directly in a component?

Reply:
Yes

Reply:
Thanks